PSPs/drafts/psp-22.md

```diff
@@ -831,8 +819,8 @@ type Balance = u128;
 ```
 
 ### Errors
-Suggested methods don't return `Result` (except `before_received`). Instead, they panic.
-This panic must be "revert with message" and  can contain one of the following messages:
+Suggested methods returns `Result`.  
```

Comment:
Suggested methods revert the transaction and return `Result` with error from this list: